src/components/atoms/error/error.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Error = ({ error }) => <p>{error.message}</p>;
```

Comment:
Simple error renderer for server based errors. Definitely doesn't cover all usages and cases but gets the job done for this PR. Definitely should be revisited and maybe moved to molecules since the error can be in different states.